pyspline/pyCurve.py

```diff
@@ -33,9 +33,9 @@ class Curve(object):
     Parameters
     ----------
     k : int
```

Comment:
There is specific numpydoc style for indicating that a parameter can only take in a few choices. See [here](https://numpydoc.readthedocs.io/en/latest/format.html#parameters)